docs/examples/workflows/script_artifact_passing.md

```diff
@@ -51,6 +48,7 @@
               artifacts:
               - from: '{{steps.generate-artifact.outputs.artifacts.hello-art}}'
                 name: message
+                path: /tmp/hello_world.txt
```

Comment:
Maybe only `name` and `from` should be copied over, thoughts here @elliotgunton ?

Reply:
Adjusted, please see https://github.com/argoproj-labs/hera/pull/647/files#diff-079dae561586ca661f24348fd6560e265e3c6eba6650abd05491201598ae9288R537

Reply:
Sounds right to me 💯